packages/namekit-react/src/components/TruncatedText.tsx

```diff
@@ -22,25 +22,25 @@ import { Tooltip } from "./Tooltip";
 interface TruncatedTextProps {
   text: string;
   maxDisplayWidth: number;
-  maxTooltipWidth?: number;
   textStylingClasses?: string;
-  tooltipTextStylingClasses?: string;
   displayTooltipWhenTextOverflows?: boolean;
+  maxTooltipWidth?: number;
+  tooltipTextStylingClasses?: string;
```

Comment:
I think all text related props should come before tooltip related props. So I updated the order.